wasm/wasi/wit/deps/hermes-logging/api.wit

```diff
@@ -4,11 +4,11 @@
 ///
 /// ## Permissions
 ///
-/// This API is ALWAYS available, and not permissioned.
+/// This API is ALWAYS available, and not permissionless.
```

Comment:
Quite confuse, try avoid double negation

Reply:
i made all the permission comments the same.  And simpler.